prac_05/colour_names.py

```diff
@@ -0,0 +1,12 @@
+COLOUR_CODES = {"AliceBlue": "#f0f8ff", "AntiqueWhite": "#faebd7", "AntiqueWhite1": "#ffefdb",
+                "AntiqueWhite2": "#eedfcc", "AntiqueWhite3": "#cdc0b0", "AntiqueWhite4": "#8b8378",
+                "aquamarine1": "#7fffd4"}
+
+colour = input("Enter colour name: ")
+while colour != "":
+    if colour in COLOUR_CODES:
+        print("The hexadecimal code for", colour, "is", COLOUR_CODES[colour])
+    else:
+        print("Invalid colour name")
```

Comment:
maybe make this something like "colour is not in colour dictionary" because I can type blue and it will say invalid colour name.